coc/client.py

```diff
@@ -329,10 +329,10 @@ async def login_with_tokens(self, *tokens: str) -> None:
         tokens: list[str]
             Tokens as found from https://developer.clashofclans.com under "My account" -> <your key> -> "token".
         """
+        self.correct_key_count = len(tokens)
```

Comment:
please update `self.key_count` too. Just to keep everything consistent.

Reply:
Done!